modules/apps/product-navigation/product-navigation-taglib/src/main/resources/META-INF/resources/control_menu/page.jsp

```diff
@@ -18,7 +18,9 @@ ProductNavigationControlMenuTagDisplayContext productNavigationControlMenuTagDis
 		<liferay-util:dynamic-include key="com.liferay.product.navigation.taglib#/page.jsp#pre" />
 
 		<div class="control-menu control-menu-level-1 control-menu-level-1-<%= applicationsMenuApp ? "light" : "dark" %> d-print-none" data-qa-id="controlMenu" id="<portlet:namespace />ControlMenu">
-			<clay:container-fluid>
+			<clay:container-fluid
+				size='<%= !FeatureFlagManagerUtil.isEnabled("LPS-184404") ? "xl" : "fluid-layout" %>'
+			>
```

Comment:
I think this may be cleaner if we use the `<clay:container>` tag instead. Passing fluid-layout to size is going to result in an extra classname ("container-fluid-max-fluid-layout") added that does nothing. If we use the `<clay:container>` tag, it could look like this:
```
<clay:container fluid="<%= true %>">
```

So either that, or we could add a new parameter to the container-fluid tag ('fullWidth'), and use that property to not set the size attribute so that we don't print an extra className.

Reply:
Ooh... better yet, update the tag to handle size the same way ClayLayout.ContainerFluid does. Make it so passing in false to size will also keep from setting the size. Then you can update this to match your other usages:
```
<clay:container-fluid
	size='<%= FeatureFlagManagerUtil.isEnabled("LPS-184404") ? false : "xl" %>'
>
```

Reply:
`size` is defined as a String in the `frontend-taglib-clay/.../ContainerFluidTag.java`, we get a build error because boolean. I had to use `"false"`.

Reply:
Okay... I think handling a boolean as a string like this creates some code smell. Instead, it's probably better if we add a new attribute to the container-fluid tag: `fullWidth`. We could add a check for it [here](https://github.com/liferay/liferay-portal/blob/master/modules/apps/frontend-taglib/frontend-taglib-clay/src/main/java/com/liferay/frontend/taglib/clay/servlet/taglib/ContainerFluidTag.java#L23) like this:
```
if (Validator.isNull(getSize()) && !getFullWidth()) {
```

This would allow our usages to look like this:
```
<clay:container-fluid
	fullWidth='<%= FeatureFlagManagerUtil.isEnabled("LPS-184404") %>'
>
```